datajoint/external.py

```diff
@@ -276,11 +276,11 @@ def references(self):
         """
         :return: generator of referencing table names and their referencing columns
         """
-        return self.connection.query("""
+        return [{k.lower(): v for k, v in elem.items()} for elem in self.connection.query("""
```

Comment:
replace list comprehension with a generator expression just as the docstring specifies.

Reply:
Good point, I have updated it to be a proper generator